src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -130,7 +130,9 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode): void
                 ->scalarNode('autoescape_service')->defaultNull()->end()
                 ->scalarNode('autoescape_service_method')->defaultNull()->end()
                 ->scalarNode('base_template_class')->example('Twig\Template')->cannotBeEmpty()->end()
-                ->scalarNode('cache')->defaultValue('%kernel.cache_dir%/twig')->end()
+                ->scalarNode('cache')
+                    ->setDeprecated('symfony/twig-bundle', '7.1', 'Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.')
```

Comment:
what will happen with the `twig.template_cache_warmer` service which is currently removed when cache is set to `false`?

Reply:
Oh good catch, I missed the possibility to set  `cache` to false. I propose to do the same as with the router, add a configuration that can only enable/disable the cache for twig, to preserve that feature, while removing the ability to change the location of the twig cache.
Would that work for you ?

Reply:
I went with another way. I only deprecated setting `cache` to a string, keeping the ability to set it to `false`, so the behaviour is preserved.